config/routes.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  namespace :api, defaults: { format: 'json' } do
+    namespace :v1 do
+      resources :topics
```

Comment:
`resources :topics` generates routes for index, show, destroy, etc... only index is implemented
You can specify that!

Reply:
add in routes
`resources :topics, only: :index`